google-cloud-secret_manager/lib/google/cloud/secret_manager.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google-cloud-secret_manager"
+
+module Google
+  module Cloud
+    ##
+    # TODO
+    module SecretManager
+      ##
+      # TODO
+      def self.new version: :v1beta1, service: :secret_manager_service, &block
```

Comment:
What we are calling `version` here is really the protobuf `package`. But the term we have used for the protobuf `service` is `client`. Should the `service` named argument be changed to `client`?

`package` : `service` :: `version` : `client`

Reply:
I think the keyword names work as is. They reflect what a customer would call these concepts ("version" and "service").